### DISCUSSION TITLE
Release 1314 Testing

### DISCUSSION THREAD

**Discussion by badosu** [beyond-all-reason/RecoilEngine#485] [1. General Discussion]
Reports for 1314 testing, only bugreports or breaking changes.

**Comment by badosu** [comment]
- Minimap:
  - ~Issuing commands on minimap does not work if cursor is over void (behind minimap)~ (Fixed on #488)
  - ~Not drawing selection box~ (Fixed on https://github.com/beyond-all-reason/spring/commit/99ca831606debfdd3524b2d5251b2c48f9ab73dc)
  - ~Bugged drawing of build commands proxying from minimap to map~ (Fixed on https://github.com/beyond-all-reason/spring/pull/488)

**Reply by lhog** [reply]
Bumpwater:
* ~~No Waves as reported by Moose~~ ?
* ~~No support for resolution changes~~

**Comment by badosu** [comment]
All issues (listed here) addressed and released
